_component_design/text-inputs.md

```diff
@@ -13,185 +13,291 @@ last-modified: 2017-08-17
+
+#### Are non-required fields always necessary?
+ * Design goal: Enable users to get what they need with as few up-front choices as possible
+ * In our industry, complex resources often have fields that are configurable AFTER the resource is created  
+ * Consider only including required fields in the main create form, and placing non-required fields in an optional, post-creation workflow
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/text-fields/text-fields-text-input-states-overview.svg)
+{% endfigure %}
+
+{% endcolumn %}
 </div>
 
-<!-- Submitting Type Ahead as a separate story
-        <div class="formGroup">
-          <div class="formLabel">Type Ahead</div>
-          <div class="formComponant">
-            <div class="ui-widget">
-              <input id="tags" type="text">
-            </div>
-          </div>
-        </div>
-
-        <div class="formGroup">
-          <div class="formLabel">Type Ahead Multiple Values</div>
-          <div class="formComponant">
-            <div class="ui-widget">
-              <input id="tags-multi" type="text">
-            </div>
-          </div>
-        </div>
--->
-
-## Disabled
-
-<div class="ui form">
-  <div class="disabled field">
-    <label>Disabled with value</label>
-    <input type="text" value="Value text" disabled>
-  </div>
-  <div class="disabled field">
-    <label>Disabled with placeholder text</label>
-    <input type="text" placeholder="Placeholder text" disabled="disabled">
-  </div>
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Disabled state
+ * Disabled text fields are not editable. They have a slightly darker background fill, lighter border, and lighter input text
+ * Disabled fields are also indicated with a cursor change to “not-allowed”, reinforcing the state for the user
+ * When a field is disabled, consider changing the help text so the user understands why it is disabled
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/text-fields/text-fields-text-input-disabled.svg)
+{% endfigure %}
+
+{% endcolumn %}
 </div>
 
-## Completed
-<div class="ui form success">
-  <div class="field">
-    <label>Complete form field</label>
-    <input type="text" value="Value text">
-    <div class="ui success message icon">
-      <i class="check circle icon"></i>
-    </div>
-  </div>
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Focused
+ * When a text field is selected, the border color changes to Cyan 700, and is raised by a shadow to give the user an indication of where they are in the form
```

Comment:
I'd personally put the color code in code as I'd be looking at the color from a code standpoint here:
```markdown
color changes to `cyan 700` and is raised
```
However, I think that perspective can be argued.
Note the removed comma.

Reply:
👍